spec/spec_tests/data/sdam/rs/secondary_wrong_set_name_with_primary_second.yml

```diff
@@ -0,0 +1,73 @@
+description: "Secondary wrong setName with primary when secondary responds first"
```

Comment:
Has this been merged into the specifications repo yet?

Reply:
This change is not wanted in specs, see my PR there.

Reply:
Ah, I see Jesse's comment now. Thanks for pointing me to it!